src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -82,10 +86,18 @@ private <T> List<T> runSnapshotBuilderQuery(
       BigQueryDatasetPdao.Converter<T> bqConverter,
       AzureSynapsePdao.Converter<T> synapseConverter) {
     String sql = query.renderSQL(createContext(dataset, userRequest));
-    return CloudPlatformWrapper.of(dataset.getCloudPlatform())
-        .choose(
-            () -> bigQueryDatasetPdao.runQuery(sql, dataset, bqConverter),
-            () -> azureSynapsePdao.runQuery(sql, synapseConverter));
+    Instant start = Instant.now();
+    List<T> result =
+        CloudPlatformWrapper.of(dataset.getCloudPlatform())
+            .choose(
+                () -> bigQueryDatasetPdao.runQuery(sql, dataset, bqConverter),
+                () -> azureSynapsePdao.runQuery(sql, synapseConverter));
+    logger.info(
```

Comment:
Until we have queries debugged and working well, I think it would be helpful to log queries as they're run along with the time each query takes.

Reply:
Do we want to add a ticket for the future to remove this logging?

Reply:
I think we can wait to remove it post-mvp, so I would wait on adding a ticket to remove it. Converting it from `info` to `debug` may be sufficient too.